configure.ac

```diff
@@ -231,6 +231,12 @@ LIBMESH_TEST_CXX11_NOEXCEPT
 #-----------------------------------------------------
 LT_INIT([disable-static])
 
+# We need to set the no-undefined flag for linking 
+# with libtool, because Windows requires all shared
+# libraries to include all used symbols. This flag
+# should do no harm on other platforms.
+libmesh_LDFLAGS+="-no-undefined"
```

Comment:
Oh I see. The INL testing didn't make it past `make check` in libmesh, so we may have eventually had the same problem...